doc/devdocs/readme.md

```diff
@@ -66,10 +41,11 @@ Various tools used by PowerToys. Includes the Visual Studio 2019 project templat
 2. Visual Studio Community/Professional/Enterprise 2019
 3. Once you've cloned and started the `PowerToys.sln`, in the solution explorer, if you see a dialog that says `install extra components`, click `install`
 
-**Optional step:**<br/>
-4. to build the Video Conference module, install the [WDK version 1903](https://docs.microsoft.com/en-us/windows-hardware/drivers/other-wdk-downloads) ([direct download link](https://go.microsoft.com/fwlink/?linkid=2085767))<br />
-   During the installation, make sure that, when prompted, the `Install Windows Driver Kit Visual Studio extension` option is checked.
-
+### Get Submodules to compile
+We have 
```

Comment:
Do you want to describe that we have some submodules and dependency to other repos that have to be initialized befor building PT?